### ISSUE TITLE
Suggestion-Per site enable

### ISSUE THREAD

**Issue by PurpleHel** [Bytezz/ghostbuster-addon#1]
I'd really like to have the option to whitelist which sites this extension works on, as I only want it to do the thing on my security camera monitoring, not regular webpages.
Do you have any plans to add such a feature? The extension works great otherwise!

**Comment by Bytezz** [comment]
How the heck did you find this addon?  
This is now basically abandonware, but I'll look into it anyway.

**Comment by PurpleHel** [comment]
Searched Firefox extension prevent burn in? It was the first reasonable looking link in the results. But if you're not developing it anymore/updating it is a hassle I can look for another one.


I set up an extra monitor on my desktop for watching my security/cat cam when I noticed faint burn in on my tablet, and then a couple days later I remembered that monitor is like 15 years old and probably even more subject to burn in than my 2020 tablet, and that led here. :D
